pages/preprints.py

```diff
@@ -94,6 +96,7 @@ class PreprintDiscoverPage(BasePreprintPage):
 
     # Group Locators
     search_results = GroupLocator(By.CSS_SELECTOR, '.search-result h4 > a')
+    no_results = GroupLocator(By.CSS_SELECTOR, 'div[class="text-muted text-center"]')
```

Comment:
This selector seems imprecise.  I could imagine something unrelated on this page having the same properties (like a copyright statement, license, etc.)  Are there any semantically meaningful classes/attributes you can use to identify this?  Is this div inside a `.search-results` container?

Reply:
Great point! I've used a more specific class `'.search-results-section .text-muted'` to search for the "No results text message"